examples/stress-nodes.js

```diff
@@ -0,0 +1,24 @@
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+import { StressNodesAttack } from '../src/stress.js';
+
+export default function () {
+  const k8sClient = new Kubernetes()
+  const stress = new StressNodesAttack(k8sClient); 
+  const nodeNames = k8sClient.nodes.list().map(node => node.name)
+  
+  // stress the nodes of the cluster
+  stress
+    .withCPU("100")
+    .withIO("100")
+    .withVM("100")
+    .withTimeout("15s")
+    .inNodes(nodeNames)
```

Comment:
Thoughts on passing Node objects instead of nodeNames?

Reply:
Depends on the usa case. If the purpose is to stress the node(s) on which a particular pod is running, then the node would come from the Pod object and we only have the name. We can always use the `nodes` api to retrieve the node object, but that makes the client logic more complex.